### PR TITLE
docs: ORPHANED Vnet Module

### PR DESCRIPTION
**Pull request by elbatane**
## Description

VNET module Orphaned

## Pipeline Reference

| Pipeline |
| -------- |
|    [avm.res.network.virtual-network](https://github.com/Azure/bicep-registry-modules/actions/workflows/avm.res.network.virtual-network.yml)      |

## Type of Change

- [ ] Update to CI Environment or utilities (Non-module affecting changes)
- [ ] Azure Verified Module updates:
  - [ ] Bugfix containing backwards-compatible bug fixes, and I have NOT bumped the MAJOR or MINOR version in `version.json`:
    - [ ] Someone has opened a bug report issue, and I have included "Closes #{bug_report_issue_number}" in the PR description.
    - [ ] The bug was found by the module author, and no one has opened an issue to report it yet.
  - [ ] Feature update backwards compatible feature updates, and I have bumped the MINOR version in `version.json`.
  - [ ] Breaking changes and I have bumped the MAJOR version in `version.json`.
  - [X] Update to documentation

## Checklist

- [ ] I'm sure there are no other open Pull Requests for the same update/change
- [ ] I have run `Set-AVMModule` locally to generate the supporting module files.
- [ ] My corresponding pipelines / checks run clean and green without any errors or warnings

### CODE REVIEW

File: avm/res/network/virtual-network/main.json [hunk 1]
```diff
@@ -5,8 +5,8 @@
   "metadata": {
     "_generator": {
       "name": "bicep",
-      "version": "0.29.47.4906",
-      "templateHash": "17645744687566805489"
+      "version": "0.21.1.54444",
```
Comment (eriqua) [inline avm/res/network/virtual-network/main.json:8]: Hey @elbatane please don't downgrade the bicep version. 
You can avoid to recompile the json as the update only affects the readme file

**Review by eriqua — COMMENTED** [review]
💔

File: avm/res/network/virtual-network/main.json (hunk 1 above)
Comment (elbatane) [reply avm/res/network/virtual-network/main.json]: Done!

File: avm/res/network/virtual-network/main.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
     "_generator": {
       "name": "bicep",
       "version": "0.29.47.4906",
-      "templateHash": "17645744687566805489"
+      "templateHash": "15637892116799403398"
```
Comment (eriqua) [inline avm/res/network/virtual-network/main.json:9]: I still see the template hash changes. Could this please be reverted too? The readme update should not impact this

Comment (elbatane) [reply avm/res/network/virtual-network/main.json]: done

**Review by eriqua — APPROVED** [review]
🆗

**Pull request merged**